src/lib/common/Header.svelte

```diff
@@ -10,6 +13,15 @@
 
 <div class="green-bar" />
 
+{#if appVersion() != "Public Beta" && appVersion() != "Private (development)"}
+  <WarningText>
+    NB this is a testing environment. Should you be using
```

Comment:
I'd still change the wording, see my other comment. At minimum, let's avoid "NB"